go.mod

```diff
@@ -1,75 +1,80 @@
 module github.com/topolvm/pie
 
-go 1.24.0
+go 1.24.9
+
+toolchain go1.24.11
```

Comment:
According to our policy we shouldn't use the `toolchain` directive. `go 1.24.0` will suffice.